packages/block-library/src/post-excerpt/edit.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * WordPress dependencies
+ */
+import { useEntityProp, useEntityId } from '@wordpress/core-data';
+import { RichText } from '@wordpress/block-editor';
+
+function PostExcerptDisplay() {
+	const [ excerpt ] = useEntityProp( 'postType', 'post', 'excerpt' );
+	return <RichText.Content tagName="p" value={ excerpt } />;
```

Comment:
In the sidebar of the editor, we use a TextareaControl to edit the excerpt, it makes me think we should use PlainText here. any reason for not making this editable?

Reply:
I was trying to be minimal with these changes by making the new blocks read-only first so that we can land them faster and then iterate.

I've changed it to use `PlainText` now since I agree it is more fitting, and the complexity doesn't increase much.